src/sidebar/components/MarkdownView.tsx

```diff
@@ -35,21 +76,49 @@ export default function MarkdownView({
     });
   }, [markdown]);
 
+  useEffect(() => {
+    const listenerCollection = new ListenerCollection();
+    const foundMentions = replaceMentionTags(content.current!, mentions);
+
+    for (const [mention, element] of foundMentions) {
+      listenerCollection.add(element, 'mouseenter', () => {
+        setActiveMention(mention);
+        mentionsPopoverAnchorRef.current = element;
+      });
+      listenerCollection.add(element, 'mouseleave', () => {
+        setActiveMention(null);
+        mentionsPopoverAnchorRef.current = null;
+      });
+    }
```

Comment:
You don't need a per-element listener here. You can use listeners on the container element which catches bubbling events.